package.json

```diff
@@ -57,10 +57,11 @@
     "serve:client": "cd ./client && npm run serve",
     "start": "npm-run-all ensure-env -p develop:server serve:client",
     "test": "npm-run-all -p test:*",
-    "test:client": "cd ./client && npm test && cd ../",
-    "test:curriculum": "cd ./curriculum && npm test && cd ../",
+    "test:client": "cd ./client && npm test",
+    "test:curriculum": "cd ./curriculum && npm test",
+    "test-curriculum-full-output": "cd ./curriculum && npm run test:full-output",
```

Comment:
```suggestion
    "test:curriculum-full-output": "cd ./curriculum && npm run test:full-output",
```

Reply:
It is intentional, we could use scripts with a tool like [`npm-run-all`](https://www.npmjs.com/package/npm-run-all)

If you explore the root package.json and the scripts in there, you will find some examples.

Reply:
Right. Thank you, for the clarification.